Sources/NIOHTTP2/MultiplexerAbstractChannel.swift

```diff
@@ -23,9 +23,11 @@ import NIOCore
 ///
 /// Note that while this is a `struct`, this `struct` has _reference semantics_.
 /// The implementation of `Equatable` & `Hashable` on this type reinforces that requirement.
+@usableFromInline
 struct MultiplexerAbstractChannel {
-    private(set) var baseChannel: HTTP2StreamChannel
+    @usableFromInline var _baseChannel: HTTP2StreamChannel
```

Comment:
Might be mistaken but I _think_ you can apply `@usableFromInline` to a `private(set) var` (but not a `private var`)